doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -0,0 +1,563 @@
+
+IronSide SE update status
+=========================
+
+The Secure Domain boot ROM code (SDROM) reports the status of an IronSide SE update request through ``SICR.UROT.UPDATE.STATUS``.
+The value of this register is copied to the IronSide SE update status field of the boot report.
+
+UICR error description
+======================
+
+This field indicates if any UICR error occurred.
+
+Local domain context
+====================
+
+This field is populated by the local domain that is invoking the CPUCONF service.
+It is set to `0` for the application core which is booted by IronSide SE.
+This service is used when one local domain boots another local domain.
+The caller can populate this field with arbitrary data that will be made available to the local domain being booted.
+Tipical examples of data that could be passed include IPC buffer sizes or the application firmware version.
+The unused parts of this field are set to 0.
+
+Random data
+===========
+
+This field is filled with random data generated by a CSPRNG.
+This data is suitable as a source of initial entropy.
+
+.. _ironside_se_booting:
+
+Booting of other domains
+************************
+
+IronSide SE boots the System Controller core first, followed by the application core, in that order.
+When booting the application core, IronSide SE does the following:
+
+* Sets ``CPUCONF[APP].INITSVTOR`` to the first 32-bit word of the application-owned memory.
+* Reads the reset vector from the second 32-bit word of the application-owned memory.
+* If the reset vector is set to ``0xFFFFFFFF``, sets ``CTRL_AP.BOOTSTATUS.BOOTERROR`` to indicate that no firmware is programmed.
+* If any other error is encountered during initialization, sets ``CTRL_AP.BOOTSTATUS.BOOTERROR`` accordingly.
+* If ``CTRL_AP.BOOTSTATUS.BOOTERROR`` is non-zero, sets ``CPUCONF[APP].CPUWAIT`` to ``1``; otherwise, sets it to ``0``.
+* Sets ``CPUCONF[APP].CPUSTART`` to ``1``.
+
+UICR Error handling
+===================
+
+If an invalid ``UICR`` configuration is detected, IronSide SE performs the following procedure:
+
+* Stops the allocation procedure.
+* Updates the boot report to indicate the ``UICR`` entry (and, if applicable, the array index) that triggered the failure.
+* Sets ``CTRL_AP.BOOTSTATUS.BOOTERROR`` to indicate the source of the error.
+* Starts the application core with ``CPUCONF[APP].CPUWAIT = 1`` (halted mode).
+
+This allows the error report to be read by a debugger, if the device is not protected.
+
+.. _ug_nrf54h20_ironside_se_cpuconf_service:
+
+IronSide SE CPUCONF service
+***************************
+
+The ``IronSide SE CPUCONF`` service enables the application core to trigger the boot of another CPU at a specified address.
+
+Specifically, IronSide SE sets ``NRF_CPUCONF_S->INITSVTOR`` with the address provided to the IronSide call, and then writes ``0x1`` to ``NRF_RADIOCORE_CPUCONF_S->CPUSTART`` to start the target CPU.
```

Comment:
We should align the notation used for CPUCONF. Neither NRF_CPUCONF_S nor NRF_RADIOCORE_CPUCONF_S is currently defined by the MDK when building for the application core, so it might be better to use "the CPUCONF instance of the processor being booted".

Which processor to boot is configurable in the service call, so we shouldn't limit it to RADIOCORE here.